src/renderer/components/input/validators/is-path.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { AsyncInputValidationError, inputValidator } from "../input_validators";
+import pathExistsInjectable from "../../../../common/fs/path-exists.injectable";
+
+const isPathInjectable = getInjectable({
+  id: "is-path",
+
+  instantiate: (di) => {
+    const pathExists = di.inject(pathExistsInjectable);
+
+    return inputValidator<true>({
+      debounce: 100,
+      condition: ({ type }) => type === "text",
+
+      validate: async (value) => {
+        try {
+          await pathExists(value);
+        } catch {
+          throw new AsyncInputValidationError(
+            `${value} is not a valid file path`,
+          );
+        }
+      },
```

Comment:
This is wrong, as seen in #5403 `pathExists` never throws.

Reply:
Hm, we've just made it injectable to get rid of usage of global shared state.

Reply:
Sure. Async validators are also causing trouble in unit tests so there is lot to improve. But, as you said we can tackle that in separate PR. :)